contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
i can definetly just "not" commit my changes, but then `vagrant up` will fail every time.  there are **alot** of parameters in that one file, which means almost every user will have to generate or delete or sed replace the file .